lapis/src/main/kotlin/org/genspectrum/lapis/model/AdvancedQueryCustomListener.kt

```diff
@@ -41,6 +42,14 @@ import java.time.LocalDate
 import java.time.format.DateTimeParseException
 import java.util.Locale
 import kotlin.collections.ArrayDeque
+import kotlin.collections.Map
```

Comment:
Are these imports actually needed? I don't see any code changes in this file that would require them

Reply:
That's weird, thanks for catching this. I don't know how they ended up there...